.cspell.json

```diff
@@ -5,7 +5,6 @@
   "language": "en",
   // words - list of words to be always considered correct
   "words": [
-    "addonsecurity",
```

Comment:
Being an id we could use backtick-formatting (which is ignored) rather than adding an exception on the word :)

Reply:
Correct :)